common/shlibs

```diff
@@ -4164,3 +4164,4 @@ liblowdown.so lowdown-0.11.2_1
 libjaylink.so.0 libjaylink-0.2.0_1
 libnvidia-container.so.1 libnvidia-container-1.10.0_1
 libnvidia-container-go.so.1 libnvidia-container-1.10.0_1
+libguestfs.so.0 libguestfs-tools-1.48.0_1
```

Comment:
This conflicts with the `libguestfs` package, is that correct

Reply:
> This conflicts with the `libguestfs` package, is that correct?

That seems to be a fault of mine, but there is some weirdness here.  There is no `libguestfs.so.*` in this package.  I mindlessly added it, because the build (the pkg phase) complained about it(!)  It looks the source of the complaint is that makedepends pulls `libguestfs-devel` in, with this library:

   SONAME: libguestfs.so.0 <-> UNKNOWN PKG PLEASE FIX!

So it would seem the complaint is about the lack of package mapping for the library, to let xbps know where the library comes from, right?  It's just weird to point to another package which is not in this change.